templates/level-page.html

```diff
@@ -75,12 +75,12 @@ <h2 class="px-4 pt-2">{{_('commands')}}</h2>
                   <label for="keyword_toggle" class="flex flex-row items-center justify-center cursor-pointer"
                     data-cy="kwlang-switch-toggle">
                     <div class="relative flex w-full">
-                      <input id="keyword_toggle" type="checkbox" {% if ((g.keyword_lang !="en" and not
+                      <input id="keyword_toggle" type="checkbox" {% if ((g.keyword_lang != "en" and not
                         request.args.get('keyword_language')) or (request.args.get('keyword_language') and
-                        request.args.get('keyword_language') !="en" )) %}checked{% endif %}
-                        onclick="hedyApp.toggle_keyword_language('{% if ((g.keyword_lang != " en" and not
+                        request.args.get('keyword_language') != "en" )) %}checked{% endif %}
+                        onclick="hedyApp.toggle_keyword_language(`{% if ((g.keyword_lang != "en" and not
```

Comment:
This part will convert to ``onclick="hedyApp.toggle_keyword_language(`en`)"``

I am not sure about the use of backticks here and can make confusion for other developers.

I suggest single quotes if you don't have a specific purpose.

Reply:
Thanks Yilmez, this amount of qoutes all over the place cuases more confusion than clarity, let's keep thinking on how to approach it more elegantly moving further. Check out my last commit :)